Chore/0.35 release branch (#1253)
Kong/docs.konghq.com: app/enterprise/0.35-x/kong-manager/authentication/basic.md

---
title: How to Enable Basic Auth for Kong Manager
book: admin_gui

> ---

Reviewer: ```suggestion
toc: false
---
```

> 1. To enable Basic Authentication, configure Kong with the following properties:

Reviewer: ```suggestion
## Prerequisites

To enable Basic Authentication, configure Kong with the following properties:
```

```
enforce_rbac = on
admin_gui_auth = basic-auth
admin_gui_session_conf = { "secret":"set-your-string-here" }
```

⚠️**Important:** the **Sessions Plugin** requries a secret and is configured securely by default. 
* Under all circumstances, the `secret` must be manually set to a string.
* If using HTTP instead of HTTPS, `cookie_secure` must be manually set to `false`. 
* If using different domains for the Admin API and Kong Manager, `cookie_samesite` must be set to `off`. 
Learn more about these properties in [Session Security in Kong Manager](/enterprise/{{page.kong_version}}/kong-manager/authentication/sessions/#session-security), and see [example configurations](/enterprise/{{page.kong_version}}/kong-manager/authentication/sessions/#example-configurations).

> 2. Start Kong:

Reviewer: ```suggestion
## Step 1

Start Kong:
```

Reviewer: (1. was Prerequisites, hence 2. is Step 1)

```
$ kong start [-c /path/to/kong/conf]
```

> 3. If you created a **Super Admin** via database migration, log in to Kong 

Reviewer: ```suggestion
## Step 2

If you created a **Super Admin** via database migration, log in to Kong 
```

Reviewer: (see above for why it's Step 2 instead of 3)
